### ISSUE TITLE
Check for send message permissions in the target channel in tul!edit command

### ISSUE THREAD

**Issue by spiralw** [Tupperbox/Tupperbox-Issues#76]

**Comment by Roryl-c** [comment]
Send message perms don't seem to be needed for the target channel in order for the edit to work so it seems like this is a non-issue

**Comment by spiralw** [comment]
webhooks can edit messages without the bot having send message permissions; users typically can't do that, so the bot should enforce an artificial restriction on it

**Comment by Roryl-c** [comment]
fair enough - in that case should we also check for the user having send message perms in the target channel (in the case of using a message link)?

**Comment by spiralw** [comment]
that's what i meant, yeah
